### DISCUSSION TITLE
Initial posture

### DISCUSSION THREAD

**Discussion by combinatorylogic** [google-deepmind/mujoco#352] [General]
Coming from Bullet, I'm having troubles setting up MuJoCo. In Bullet I'd normally drop the robot from some height, let it settle and then start controlling it. In MuJoCo joints are flexible and I could not find a way to stiffen them temporarily, so that the initial posture won't change too much after falling and settling. Any suggestions? There must be some canonical way of doing it.

**Comment by yuvaltassa** [comment]
The easiest way is to increase the armature of all the joints by a lot:
```xml
<default>
  <joint armature="1e5"/>
</default>
```

**Reply by combinatorylogic** [reply]
Thanks, worked perfectly!
